cdc/redo/common/util.go

```diff
@@ -58,7 +58,6 @@ var InitS3storage = func(ctx context.Context, uri url.URL) (storage.ExternalStor
 	s3storage, err := storage.New(ctx, backend, &storage.ExternalStorageOptions{
 		SendCredentials: false,
 		HTTPClient:      nil,
-		SkipCheckPath:   true,
```

Comment:
tidb updated to release-5.3, need to remove this and below @liuzix 

it's removed in cdc 5.4 see https://github.com/pingcap/tiflow/pull/3853/files#diff-d534645fb73a1d612df0f45b7f07dac2f2edaf7648b08b818f74fbf92e1df231